### DISCUSSION TITLE
SDC for the BCP

### DISCUSSION THREAD

**Discussion by IGNLab** [nipreps/nibabies#387] [General]
Hello,

I am trying to run SDC with the fmaps from the BCP. Are there any tips to doing so? I am getting a 'sophisticated PEPOLAR schemes' error.

Thanks!

**Comment by mgxd** [comment]
Hi - can you elaborate on the fieldmaps running into a problem? I thought BCP was using multiple phase encoding directions (pepolar), which should be handled by [sdcflows](https://www.nipreps.org/sdcflows/master/index.html)
